### PR TITLE
Additional tagged text. (500 tokens x 2)

### PR DESCRIPTION
**Pull request by nu11us**
Evolution excerpt from simple wikipedia. 
https://simple.wikipedia.org/wiki/Evolution

### CODE REVIEW

File: texts/MANIFEST [hunk 1]
```diff
@@ -3,3 +3,4 @@
 1   icarus      dolphingarlic   ACCESSED FROM: http://www.angelfire.com/dragon/erzulie/Icarus_Allsorts.htm                                      AUTHOR: Roger McGough           STYLE: Poem               LICENSE: Public Domain    YEAR: 1816
 1   kubla_khan  dolphingarlic   ACCESSED FROM: https://www.poetryfoundation.org/poems/43991/kubla-khan                                          AUTHOR: Samuel Taylor Coleridge STYLE: Poem               LICENSE: Creative Commons YEAR: 1968
 1   reggae      dolphingarlic   ACCESSED FROM: https://docs.google.com/document/d/1Rz7DNd8vFPaU_baNtyZVCIxAGUsTcamFnAxHT92AXBU/edit?usp=sharing AUTHOR: Andi Qu                 STYLE: Informative Essay  LICENSE: Creative Commons YEAR: 2018
+1   evolution   nu11us          ACCESSED FROM: https://simple.wikipedia.org/wiki/Evolution                                                      AUTHOR: N/A                     STYLE: Informative Essay  LICENSE: Creative Commons YEAR: 2018
```
Comment (wei2912) [inline texts/MANIFEST]: Please link to the exact version of the page.

File: texts/formatted_evolution.txt [hunk 2]
```diff
@@ -0,0 +1,1457 @@
...
+"<old>"
+	"old" adj sint
+"<.>"
+	"." sent
+"<By>"
+;	"by" adv
+	"by" pr
+"<studying>"
+;	"study" vblex pprs
+	"study" vblex ger
```
Comment (wei2912) [inline texts/formatted_evolution.txt]: I don't think it's a gerund, could you clarify on this? Also, not sure what subs is supposed to be?

File: texts/formatted_evolution.txt [hunk 3]
```diff
@@ -0,0 +1,1457 @@
...
+;	"layer" vblex pres p3 sg
+"<of>"
+	"of" pr
+"<rock>"
+	"rock" n sg
+;	"rock" vblex inf
+;	"rock" vblex pres
+;	"rock" vblex imp
+"<that>"
+	"that" cnjsub
```
Comment (wei2912) [inline texts/formatted_evolution.txt]: Hmm thought this would be a determiner.

File: texts/formatted_evolution.txt [hunk 4]
```diff
@@ -0,0 +1,1457 @@
...
+;	"that" det dem sg
+;	"that" prn dem mf sg
+;	"that" prn rel an mf sp
+"<make up>"
+	"make# up" vblex sep inf
```
Comment (wei2912) [inline texts/formatted_evolution.txt]: Please pick one analysis here.

File: texts/formatted_evolution.txt [hunk 5]
```diff
@@ -0,0 +1,1457 @@
...
+;	"find# out" vblex pres
+;	"find# out" vblex imp
+"<about>"
+;	"about" adv
+	"about" pr
+"<its>"
+;	"its" det pos sp
+	"prpers" det pos p3 nt sg
+"<past>"
+	"past" adj
```
Comment (wei2912) [inline texts/formatted_evolution.txt]: Should be a noun and not an adjective.

File: texts/formatted_evolution.txt [hunk 6]
```diff
@@ -0,0 +1,1457 @@
...
+"<over time>"
+	"over time" adv
+"<,>"
+	"," cm
+"<because>"
+	"because" cnjsub
+;	"because" cnjadv
+"<their>"
+	"their" det pos sp
+	"prpers" det pos p3 mf pl
```
Comment (wei2912) [inline texts/formatted_evolution.txt]: Please pick this.

File: texts/formatted_evolution.txt [hunk 7]
```diff
@@ -0,0 +1,1457 @@
...
+"<the>"
+	"the" det def sp
+"<rocks>"
+	"rock" n pl
+;	"rock" vblex pres p3 sg
+"<.>"
+	"." sent
+"<These>"
+;	"this" det dem pl
+	"this" prn dem mf pl
```
Comment (wei2912) [inline texts/formatted_evolution.txt]: Determiner, not pronoun, I think.

File: texts/formatted_evolution.txt [hunk 8]
```diff
@@ -0,0 +1,1457 @@
...
+"<remains>"
+	"remains" n pl
```
Comment (wei2912) [inline texts/formatted_evolution.txt]: Pick this.

File: texts/formatted_evolution.txt [hunk 9]
```diff
@@ -0,0 +1,1457 @@
...
+	"remain" vblex pres p3 sg
+"<are called>"
+	"be# called" vblex pres p1 pl
+;	"be# called" vblex pres p2 sg
+;	"be# called" vblex pres p2 pl
+	"be# called" vblex pres p3 pl
```
Comment (wei2912) [inline texts/formatted_evolution.txt]: Pick this

File: texts/formatted_evolution.txt [hunk 10]
```diff
@@ -0,0 +1,1457 @@
...
+"<'>"
+	"'" apos
```
Comment (wei2912) [inline texts/formatted_evolution.txt]: Pick this too, same for the closing single quote.

File: texts/formatted_evolution.txt [hunk 11]
```diff
@@ -0,0 +1,1457 @@
...
+"<and>"
+	"and" cnjcoo
+"<plants>"
+	"plant" n pl
+;	"plant" vblex pres p3 sg
+"<of>"
+	"of" pr
+"<today>"
+	"today" adv
+	"today" n sg
```
Comment (wei2912) [inline texts/formatted_evolution.txt]: Pick this.

File: texts/formatted_evolution.txt (hunk 2 above)
Comment (nu11us) [reply texts/formatted_evolution.txt]: Examples of similar gerund usage: https://portlandenglish.edu/blog/by-gerund/

File: texts/formatted_evolution.txt (hunk 5 above)
Comment (nu11us) [reply texts/formatted_evolution.txt]: yeah.

File: texts/formatted_evolution.txt (hunk 7 above)
Comment (nu11us) [reply texts/formatted_evolution.txt]: https://en.wikipedia.org/wiki/Demonstrative#Demonstrative_determiners_and_pronouns

File: texts/formatted_evolution.txt (hunk 3 above)
Comment (nu11us) [reply texts/formatted_evolution.txt]: It starts the subordinate clause: `that make up Earth's crust`. So I don't see how it could be a determiner .

Comment (ftyers) [reply texts/formatted_evolution.txt]: It's a relative pronoun. You can find out by asking a question of the subordinate clause.

"What makes up the earth's crust?" -> layers of rock, so the "that" in this case stands in for the NP the clause is modifying.

Comment (nu11us) [reply texts/formatted_evolution.txt]: Agreed. Changed as of latest commit

File: texts/MANIFEST [hunk 12]
```diff
@@ -3,3 +3,4 @@
 1   icarus      dolphingarlic   ACCESSED FROM: http://www.angelfire.com/dragon/erzulie/Icarus_Allsorts.htm                                      AUTHOR: Roger McGough           STYLE: Poem               LICENSE: Public Domain    YEAR: 1816
 1   kubla_khan  dolphingarlic   ACCESSED FROM: https://www.poetryfoundation.org/poems/43991/kubla-khan                                          AUTHOR: Samuel Taylor Coleridge STYLE: Poem               LICENSE: Creative Commons YEAR: 1968
 1   reggae      dolphingarlic   ACCESSED FROM: https://docs.google.com/document/d/1Rz7DNd8vFPaU_baNtyZVCIxAGUsTcamFnAxHT92AXBU/edit?usp=sharing AUTHOR: Andi Qu                 STYLE: Informative Essay  LICENSE: Creative Commons YEAR: 2018
+1   evolution   nu11us          ACCESSED FROM: https://pastebin.com/PUbdMj8q                                                                    AUTHOR: N/A                     STYLE: Informative Essay  LICENSE: Creative Commons YEAR: 2018
```
Comment (jonorthwash) [inline texts/MANIFEST]: So you wrote this article?  I thought I saw somewhere that it came from Wikipedia?

Comment (nu11us) [reply texts/MANIFEST]: I didn't. That is a pastebin with just the text and a link to the page in the title. See the first resolved code review.

Comment (jonorthwash) [reply texts/MANIFEST]: The `MANIFEST` should link to the original source.

File: texts/MANIFEST (hunk 1 above)
Comment (jonorthwash) [reply texts/MANIFEST]: By exact version, @wei2912 means the id in the history of that page.

**Pull request closed without merging**